backend/test/integration/experiment_api_test.go

```diff
@@ -154,11 +159,18 @@ func (s *ExperimentApiTest) TestExperimentAPI() {
 	experiment, err = s.experimentClient.Get(&params.GetExperimentParams{ID: trainingExperiment.ID})
 	assert.Nil(t, err)
 	assert.Equal(t, expectedTrainingExperiment, experiment)
-
-	/* ---------- Clean up ---------- */
-	test.DeleteAllExperiments(s.experimentClient, t)
 }
 
 func TestExperimentAPI(t *testing.T) {
 	suite.Run(t, new(ExperimentApiTest))
 }
+
+func (s *ExperimentApiTest) TearDownSuite() {
```

Comment:
Is this method used in manual process? Didn't see it called in the script.

Reply:
https://godoc.org/github.com/stretchr/testify/suite#TearDownAllSuite
it will be caused by testify

Reply:
I see. Thanks!
/lgtm
/approve